src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,19 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\n Authorization for identifier {%s} failed.' % (
```

Comment:
Ah, we missed one usage here, can we please not use `%` ?

Reply:
You seem to forget i'm not a python dev.
Hence, I have no idea what you want me to do.

Reply:
`f'\n Authorization for identifier {authzr.body.identifier} failed.'`

Reply:
can you PLEASE start using github suggestions...